build/webpack.styles.js

```diff
@@ -1,27 +1,39 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
+// import postcssPlugins from './webpack.postcss'
 
-export default {
-  dev: {
+const styleLoaders = {
+  development: {
     test: /\.css$/,
     loaders: [
       'style-loader',
-      { loader: 'css-loader', query: { modules: true } },
-      { loader: 'postcss-loader' }
+      { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+      { loader: 'postcss-loader'
+        // pending new postcss-loader release - then remove loaderOptionsPlugin
+        // options: {
+        //   plugins: function () {
+        //     return postcssPlugins
+        //   }
+        // }
+      }
     ]
   },
-  prod: {
+  production: {
     test: /\.css$/,
-    loader: ExtractTextPlugin.extract({
+    loaders: ExtractTextPlugin.extract({
       fallbackLoader: 'style-loader',
       loader: [
-        { loader: 'css-loader', query: { modules: true } },
-        { loader: 'postcss-loader' }
+        { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+        { loader: 'postcss-loader'
```

Comment:
These settings for post css don't look different from the development ones. Any reason in particular why they should be environment specific? If anything, the duplicated part could be pulled out into a variable so that it only has to be changed once, so long as it is given a good descriptive name.